2-mandatory/6-writing-tests-advanced.js

```
@@ -73,19 +85,40 @@ function formatCourseworkResult(trainee) {
     subjects: ["JavaScript", "React", "CSS"]
   }
 */
-
+test("check the output of formatCourseworkResult for passed trainee with the name Ali", () => {
+  let trainee = {
+    name: "Ali",
+    score: 49,
+    age: 33,
+    subjects: ["JavaScript", "React", "CSS"]
+  }
+  expect(formatCourseworkResult(trainee)).toEqual("Ali's coursework was marked as grade E.")
+});
 /*
   Write a test that checks the output of formatCourseworkResult when passed the following trainee:
   {
     score: 90,
     age: 29
   }
 */
-
```

These are all good. Sometimes when writing tests - we mention what we expect in the name of the test. For example: `formatCourseworkResult should return an error for trainee without name`.
Then if we accidentally make a code change that causes this test to fail - the output of the tests will give us some better information 😄